File: app/src/main/java/ru/prsolution/winstrike/mvp/models/LoginModel.java
package ru.prsolution.winstrike.mvp.models;

/**
 * Created by designer on 07/03/2018.
 */
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LoginModel {

    @SerializedName("phone")
    @Expose
    private String phone;
    @SerializedName("password")
    @Expose
    private String password;

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

}

File: app/src/main/kotlin/ru/prsolution/winstrike/domain/models/LoginModel.kt
package ru.prsolution.winstrike.domain.models

import com.squareup.moshi.Json

/**
 * Created by oleg on 07/03/2018.
 */

class LoginModel(
		@field:Json(name = "phone") val phone: String? = null,
		@field:Json(name = "password") val password: String? = null
)

